assets/admin/tour/helper.js

```diff
@@ -35,22 +35,25 @@ export function removeHighlightClasses() {
 	} );
 }
 
+let stepActionTimeout = null;
+
 /**
  * Performs step actions one after another.
  *
  * @param {Array} stepActions An array of selectors to highlight.
  */
 export async function performStepActionsAsync( stepActions ) {
 	removeHighlightClasses();
+	clearTimeout( stepActionTimeout );
```

Comment:
This is to make sure actions won't happen later after moving to different steps. Based on the `removeHighlightClasses();` call in the previous line, I supposed we can assume that the `performStepActionsAsync` will always be called, right?

Reply:
Yes I think so in general. Thanks for the improvement! Just added a comment about it here https://github.com/Automattic/sensei/pull/7538#discussion_r1526478807